scripts/tools/footprint_scripts_resistorlike.py

```diff
@@ -280,16 +280,18 @@ def makeResistorAxialHorizontal(seriesname, rm, rmdisp, w, d, ddrill, R_POW, typ
 
     # add model
     if (has3d!=0):
-        kicad_mod.append(Model(filename=lib_name + ".3dshapes/"+footprint_name+".wrl", at=x_3d, scale=s_3d, rotate=[0, 0, 0]))
+        kicad_mod.append(Model(filename="${KISYS3DMOD}/" + lib_name + ".3dshapes/"+footprint_name+".wrl", at=x_3d, scale=s_3d, rotate=[0, 0, 0]))
 
     # print render tree
     # print(kicad_mod.getRenderTree())
     # print(kicad_mod.getCompleteRenderTree())
 
     # write file
     file_handler = KicadFileHandler(kicad_mod)
-    file_handler.writeFile(footprint_name+'.kicad_mod')
-
+    directory = os.path.dirname(lib_name + ".pretty/")
+    if not os.path.exists(directory):
+        os.makedirs(directory)
+    file_handler.writeFile(lib_name + ".pretty/" + footprint_name + '.kicad_mod')

```

Comment:
Simply add a check for the output directory before actually writing the footprint. This avoids errors if the directory does not exist or if library name is changed (for example in future library split...)